### DISCUSSION TITLE
Does Rehid work on CFW programs? Such as TWLMenu++?

### DISCUSSION THREAD

**Discussion by isa56** [hax0kartik/rehid#14] [Q&A] [answered]
Hello!
I have a problem with my Select button on my O3DSXL, so Rehid is one of the only ways to play normally. The thing is, in CFW and Homebrew titles it still is getting pressed endlessly, and TWLMenu++ is my main problem here.
Should it be working on it? Is there a way to stop it?

Thank you for your attention and for your work :D

**Comment by hax0kartik** [comment, marked as answer]
Rehid cannot work with ds games. Only works with 3ds games and titles.

**Reply by isa56** [reply]
Sadge. I wonder if we could do something similar for DS mode then...
